app/elements/element-action-menu/element-action-menu.html

```diff
@@ -3,39 +3,54 @@
 <link rel="import" href="../catalog-element/catalog-element.html">
 
 <dom-module id="element-action-menu">
-  <style>
-    :host {
-      @apply(--layout);
-      @apply(--layout-center);
-      @apply(--layout-center-justified);
-      position: relative;
-      text-align: center;
-    }
+  <template>
+    <style>
+      :host {
+        @apply(--layout);
+        @apply(--layout-center);
+        @apply(--layout-center-justified);
+        position: relative;
+        text-align: center;
+      }
 
-    a, iron-icon {
-      color: rgba(0,0,0,0.6);
-    }
+      a, iron-icon {
+        font-size: 12px;
+        color: #666;
+        line-height: 56px;
```

Comment:
This is causing the buttons in the action menu to be very tall (https://github.com/Polymer/polymer-element-catalog/pull/195#issuecomment-149721782).  One option is to remove this and in `element-table.html` add `padding-top: 12px` to `#actions element-action-menu`.